LM-Ontology/ontologies/SACM.sadl

```diff
@@ -6,186 +6,186 @@ import "http://arcos.rack/PROV-S".
 
-ASSERTION
-    is a type of ARGUMENT_ASSET.
-        assertionDeclaration describes ASSERTION with values of type string.
+Assertion
+    is a type of Argument_Asset.
+        assertionDeclaration describes Assertion with values of type string.
 
-CLAIM
-    is a type of ASSERTION.
+Claim
+    is a type of Assertion.
 
-ASSERTED_RELATIONSHIP
-    is a type of ASSERTION.
-        isCounter describes ASSERTED_RELATIONSHIP with values of type boolean.
-        source describes ASSERTED_RELATIONSHIP with values of type ARGUMENT_ASSET.
+Asserted_Relationship
+    is a type of Assertion.
+        isCounter describes Asserted_Relationship with values of type boolean.
+        source describes Asserted_Relationship with values of type Argument_Asset.
         source is a type of wasDerivedFrom.
-        target describes ASSERTED_RELATIONSHIP with values of type ARGUMENT_ASSET.
+        target describes Asserted_Relationship with values of type Argument_Asset.
         target is a type of wasDerivedFrom.
-        reasoning describes ASSERTED_RELATIONSHIP with values of type ARGUMENT_REASONING.
+        reasoning describes Asserted_Relationship with values of type Argument_Reasoning.
         reasoning is a type of wasDerivedFrom.
 
-ASSERTED_CONTEXT
-    is a type of ASSERTED_RELATIONSHIP.
+Asserted_Context
+    is a type of Asserted_Relationship.
 
-ASSERTED_EVIDENCE
-    is a type of ASSERTED_RELATIONSHIP.
+Asserted_Evidence
+    is a type of Asserted_Relationship.
 
 /* Artifact Classes. */
 
-ARTIFACT_PACKAGE
-    is a type of ARTIFACT_ELEMENT.
+Artifact_Package
+    is a type of Artifact_Element.
 
-ARTIFACT_ASSET
-    is a type of ARTIFACT_ELEMENT.
+Artifact_Asset
+    is a type of Artifact_Element.
 
-PROPERTY
-    is a type of ARTIFACT_ASSET.
+Property
+    is a type of Artifact_Asset.
 
-ARTIFACT_ASSET_RELATIONSHIP
-    is a type of ARTIFACT_ASSET.
-        source describes ARTIFACT_ASSET_RELATIONSHIP with values of type ARTIFACT_ASSET.
+Artifact_Asset_Relationship
+    is a type of Artifact_Asset.
+        source describes Artifact_Asset_Relationship with values of type Artifact_Asset.
         source is a type of wasDerivedFrom.
-        target describes ARTIFACT_ASSET_RELATIONSHIP with values of type ARTIFACT_ASSET.
+        target describes Artifact_Asset_Relationship with values of type Artifact_Asset.
         target is a type of wasDerivedFrom.
 
-ARTIFACT
-    is a type of ARTIFACT_ASSET.
-        artifactVersion describes ARTIFACT with values of type string.
-        artifactDate describes ARTIFACT with values of type string.
+Artifact
+    is a type of Artifact_Asset.
+        artifactVersion describes Artifact with values of type string.
+        artifactDate describes Artifact with values of type string.
 
-SACM_ACTIVITY
-    is a type of ARTIFACT_ASSET.
-        startTime describes SACM_ACTIVITY with values of type string.
-        endTime describes SACM_ACTIVITY with values of type string.
+Activity
```

Comment:
Was this supposed to lose its `SACM_` prefix? Some stuff has the prefix and some doesn't; I don't understand the pattern.

Reply:
I don't know about pattern. But you were right about the missing `SACM_` prefix on this one. It was my copy-paste error. Good catch!